modules/nw-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-banp-metric.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="anp-banp-metrics_{context}"]
+= Metrics for AdminNetworkPolicy
+
+The following table describes 12 metrics that can support monitoring your ANP and BANP.
+
+[cols="1,1,1"]
+|===
+| Name | Labels |Explanation
+
+|`ovnkube_controller_admin_network_policies`
+|Not applicable
+|The total number of `AdminNetworkPolicy` in the cluster.
+
+|`ovnkube_controller_baseline_admin_network_policies`
+|Not applicable
+|The total number of `BaselineAdminNetworkPolicy` in the cluster. Values should be 0 or 1.
+
+|`ovnkube_controller_admin_network_policies_rules`
+|*`direction`: specifies either ingress or egress.
+* `action`: specifies either `Pass`, `Allow`, or `Deny`.
+|The total number of rules across all ANP policies in the cluster grouped by `direction` and `action`.
+
+|`ovnkube_controller_baseline_admin_network_policies_rules`
+|*`direction`: specifies either ingress or egress.
+* `action`: specifies either `Allow` or `Deny`.
+|The total number of rules across all BANP policies in the cluster grouped by `direction` and `action`.
+
+|`ovnkube_controller_admin_network_policies_db_objects`
+|`tablename`
+|The total number of OVN Northbound database (nbdb) objects in the cluster which can be either `ACL` or `Address_Set`
```

Comment:
```suggestion
|The total number of OVN Northbound database (nbdb) objects that are created by all the admin network policies in the cluster.
```
 which can be either `ACL` or `Address_Set` are the labels needs to be the value of "table_name"

Reply:
I think I fixed this, but I'm not quite sure I fully got what you wanted. Please double check.